package.json

```diff
@@ -47,7 +47,7 @@
     "url": "git://github.com/UserOfficeProject/user-office-lib.git"
   },
   "engines": {
-    "npm": ">=8.0.0",
-    "node": ">=16.0.0"
+    "npm": ">=8.5.0",
```

Comment:
@jekabs-karklins  not sure if this is really needed for some reason to be grater or equal to v8.5.0 because I have node 16.14.0 installed locally and by default it installs npm v8.3.x and now it fails.

Reply:
True, I'll set it to 8.3.1

Reply:
Great because I had issues running `npm install` today in the morning 😄